From b3d28081126829ecd1c7edb7351d1cab3bd0f9ff Mon Sep 17 00:00:00 2001
From: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Date: Sun, 2 Dec 2012 08:11:38 -0500
Subject: [PATCH] f2fs: remove unused variable

The variables node_page and page_offset are initialized but never used
otherwise, so remove those unused variables.

Signed-off-by: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
---
 fs/f2fs/dir.c  | 2 --
 fs/f2fs/file.c | 2 --
 2 files changed, 4 deletions(-)

diff --git a/fs/f2fs/dir.c b/fs/f2fs/dir.c
index 089eb67..2a20c50 100644
--- a/fs/f2fs/dir.c
+++ b/fs/f2fs/dir.c
@@ -509,13 +509,11 @@ void f2fs_delete_entry(struct f2fs_dir_entry *dentry, struct page *page,
 	}
 
 	if (bit_pos == NR_DENTRY_IN_BLOCK) {
-		loff_t page_offset;
 		truncate_hole(dir, page->index, page->index + 1);
 		clear_page_dirty_for_io(page);
 		ClearPageUptodate(page);
 		dec_page_count(sbi, F2FS_DIRTY_DENTS);
 		inode_dec_dirty_dents(dir);
-		page_offset = page->index << PAGE_CACHE_SHIFT;
 		f2fs_put_page(page, 1);
 	} else {
 		f2fs_put_page(page, 1);
diff --git a/fs/f2fs/file.c b/fs/f2fs/file.c
index 89241c5..f9e085d 100644
--- a/fs/f2fs/file.c
+++ b/fs/f2fs/file.c
@@ -30,7 +30,6 @@ static int f2fs_vm_page_mkwrite(struct vm_area_struct *vma,
 	struct page *page = vmf->page;
 	struct inode *inode = vma->vm_file->f_path.dentry->d_inode;
 	struct f2fs_sb_info *sbi = F2FS_SB(inode->i_sb);
-	struct page *node_page;
 	block_t old_blk_addr;
 	struct dnode_of_data dn;
 	int err;
@@ -50,7 +49,6 @@ static int f2fs_vm_page_mkwrite(struct vm_area_struct *vma,
 	}
 
 	old_blk_addr = dn.data_blkaddr;
-	node_page = dn.node_page;
 
 	if (old_blk_addr == NULL_ADDR) {
 		err = reserve_new_block(&dn);
-- 
1.8.2.1.422.gdcd8c09

